persistence/src/commonMain/db_user/migrations/75.sqm

```diff
@@ -0,0 +1,264 @@
+import com.wire.kalium.persistence.dao.QualifiedIDEntity;
+import com.wire.kalium.persistence.dao.asset.AssetTransferStatusEntity;
+import com.wire.kalium.persistence.dao.message.MessageEntity;
+
+CREATE TABLE MessageAssetStatus (
+    message_id TEXT NOT NULL,
+    conversation_id TEXT AS QualifiedIDEntity NOT NULL,
+    transfer_status TEXT AS AssetTransferStatusEntity NOT NULL DEFAULT 'NOT_PROCESSED',
+    FOREIGN KEY (message_id, conversation_id) REFERENCES Message(id, conversation_id) ON DELETE CASCADE ON UPDATE CASCADE,
+    PRIMARY KEY (message_id, conversation_id)
+);
+
+DROP VIEW IF EXISTS MessageAssetView;
+DROP VIEW IF EXISTS MessagePreview;
+DROP VIEW IF EXISTS MessageDetailsView;
+
+ALTER TABLE MessageAssetContent RENAME TO TempMessageAssetContent;
+
+CREATE TABLE MessageAssetContent (
+      message_id TEXT NOT NULL,
+      conversation_id TEXT AS QualifiedIDEntity NOT NULL,
+      asset_size INTEGER NOT NULL,
+      asset_name TEXT,
+      asset_mime_type TEXT NOT NULL,
+      asset_otr_key BLOB NOT NULL,
+      asset_sha256 BLOB NOT NULL,
+      asset_id TEXT NOT NULL,
+      asset_token TEXT,
+      asset_domain TEXT,
+      asset_encryption_algorithm TEXT,
+      asset_width INTEGER AS Int,
+      asset_height INTEGER AS Int,
+      asset_duration_ms INTEGER,
+      asset_normalized_loudness BLOB,
+      FOREIGN KEY (message_id, conversation_id) REFERENCES Message(id, conversation_id) ON DELETE CASCADE ON UPDATE CASCADE,
+      PRIMARY KEY (message_id, conversation_id)
+);
+
+INSERT INTO MessageAssetContent (message_id, conversation_id, asset_size, asset_name, asset_mime_type, asset_otr_key, asset_sha256, asset_id, asset_token, asset_domain, asset_encryption_algorithm, asset_width, asset_height, asset_duration_ms, asset_normalized_loudness)
```

Comment:
question: shouldn't `MessageAssetStatus` table also be populated with the statuses that we already have in the `TempMessaseAssetContent` table so that we don't lose anything?